apiserver/apiserver_test.go

```diff
@@ -1,73 +0,0 @@
-// Copyright 2016 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package apiserver_test
-
-import (
-	"io"
-	"net/http"
-	"net/url"
-	"strings"
-	"time"
-
-	jc "github.com/juju/testing/checkers"
-	"github.com/juju/worker/v4/workertest"
-	gc "gopkg.in/check.v1"
-
-	"github.com/juju/juju/apiserver"
-	apitesting "github.com/juju/juju/apiserver/testing"
-	"github.com/juju/juju/internal/testing"
-	jujutesting "github.com/juju/juju/juju/testing"
-)
-
-type apiserverSuite struct {
```

Comment:
I believe this coverage could be kept with minimal effort. Doesn't really need `ApiServerSuite`

Reply:
These should be bash tests...